civictechprojects/static/css/partials/_LogInController.scss

```diff
@@ -19,6 +19,7 @@
 .LogInController-greeting {
   margin-bottom: 27px;
   min-width: 200px;
+  text-transform: uppercase;
```

Comment:
What are the advantages of using CSS rather than setting the text as explicitly uppercase?